sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -43,6 +43,7 @@ const (
 	URNReshuffle     = "beam:transform:reshuffle:v1"
 	URNCombinePerKey = "beam:transform:combine_per_key:v1"
 	URNWindow        = "beam:transform:window_into:v1"
+	URNTruncate      = "beam:transform:sdf_truncate_sized_restrictions_v1"
```

Comment:
Please move this to adjacent to `URNRequiresSplittableDoFn`, since it's associated with that feature.  It should probably be added to the capabilities requirement of the pipeline whenever there's a ProcessContinuation DoFn (see how Danny did BundleFinalization, and how the SDF one is handled.  

That can happen in this PR, or in Ritesh's, either's fine.

Reply:
Moved, we can handle adding it when Ritesh implements the truncate restriction feature